### PR TITLE
Chart Legends docs

### PR DESCRIPTION
**Pull request by svdimitr**

### CODE REVIEW

File: components/chart/legend.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+title: Legend
+page_title: Chart Legend
+description: Discover the possibilities of the Legend for the Blazor Chart. Learn how to show it, and explore the customization options.
```
Comment (dimodi) [inline components/chart/legend.md:4]:
```suggestion
description: Discover the capabilities of the Chart Legend in Telerik UI for Blazor. Learn how to show it, and explore the customization options.
```

File: components/chart/legend.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+slug: chart-legend
+tags: telerik,blazor,chart,legend,customizations
+published: True
+position: 100
+---
+
+# Telerik Chart Legend
+
+The Telerik Chart for Blazor has the capability to showcase a visual guide offering details about the elements depicted in the chart. This article explores how to add a Chart legend, identify its building blocks, and customizing the legend.
```
Comment (dimodi) [inline components/chart/legend.md:13]:
```suggestion
The Telerik Chart for Blazor can show a visual guide with details about the series or elements in the Chart. This article explores how to add a Chart legend, identify its building blocks, and customize the legend appearance.
```

File: components/chart/legend.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+## Adding a Legend
+
+1. Add the `<ChartLegend>` child tag and set the `Visible` parameter to `true`
+1. Add the `Name` parameter to all Chart series that must be visible in the legend.
+
+## Chart Legend Customization
+
+You can customize the legend of the Telerik Chart for Blazor by adding nested (child) tags to the `<ChartLegend>` and use their parameters for fine-tuning.
```
Comment (dimodi) [inline components/chart/legend.md:22]: No need to use the full component name, except in 1-2 places at the top.

```suggestion
You can customize the Chart legend by adding nested (child) tags to the `<ChartLegend>` and use their parameters for fine tuning. 
```

File: components/chart/legend.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+The structure of the nested tags is `<ChartLegend*Specifics*>`, where the specifics can be:
+
+* `Title`
+* `Item`
+* `Border`
+
+>note Use the IntelliSense to explore the nested tags.
+
+## Chart Legend Series Customization
```
Comment (dimodi) [inline components/chart/legend.md:32]: This heading doesn't make it clear what is the relationship between the series and the legend. Consider something like

```suggestion
## Legend Settings in the Chart Series
```

File: components/chart/legend.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+The structure of the nested tags is `<ChartLegend*Specifics*>`, where the specifics can be:
+
+* `Title`
+* `Item`
+* `Border`
+
+>note Use the IntelliSense to explore the nested tags.
+
+## Legend Settings in the Chart Series
```
Comment (ikoevska) [inline components/chart/legend.md:32]: Consider `## Customizing Individual Items in the Chart Legend` instead. Also would Styling fit better here?

File: components/chart/legend.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+@code {
+    public List<object> series1Data = new List<object>() { 10, 2, 5, 6 };
+    public List<object> series2Data = new List<object>() { 5, 8, 2, 7 };
+    public string[] xAxisItems = new string[] { "Q1", "Q2", "Q3", "Q4" };
+}
+````
+
+## See Also
+
+[Live Chart Legend Customization Demo](https://demos.telerik.com/blazor-ui/chart/legend-customization)
```
Comment (ikoevska) [inline components/chart/legend.md:93]: Are there relevant links to the API that you can include?

File: components/chart/legend.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+# Telerik Chart Legend
+
+The Telerik Chart for Blazor can show a visual guide with details about the series or elements in the Chart. This article explores how to add a Chart legend, identify its building blocks, and customize the legend appearance.
+
+## Adding a Legend
+
+1. Add the `<ChartLegend>` child tag and set the `Visible` parameter to `true`
+1. Add the `Name` parameter to all Chart series that must be visible in the legend.
+
+## Chart Legend Customization
```
[the remark this answers is not in the document]
Comment (ikoevska) [reply components/chart/legend.md:20]: Also, would Styling be a better term here?

**Review by ikoevska — COMMENTED** [review]
Ping @svdimitr and @dimodi When you have the time, consider going through the notes and the suggestions inline.

Also, you might want to go through some examples of docs about Chart Legends:
* https://blazor.radzen.com/chart-legend (super basic, but note how it discusses the defaults and what the legend displays; it also discusses how to show or hide the legend)
* https://blazor.syncfusion.com/documentation/chart/legend (a great example of what the Legend article should cover - it's likely that readers are interested in how to tackle position and alignment, how to show or hide the legend, and how to highlight data in the chart by selecting something in the legend)
* https://mui.com/x/react-charts/legend/ (although a React library, again, note the use cases covered by the page - it discusses showing and hiding the legend, customizing the position, styling labels, etc.)

For a future update of this page, consider covering some of the missing information. For example, is the legend enabled or disabled, by default? If not enabled, how do you enable it? If shown by default, can you hide it? Can you change its position and alignment, etc.

**Pull request merged**